### ISSUE TITLE
CMake googletest-download failed

### ISSUE THREAD

**Issue by toor1245** [google/cpu_features#201]
At the moment, all tests fail due to
```ini
...
Cloning into 'googletest-src'...
fatal: invalid reference: master
CMake Error at googletest-download/googletest-prefix/tmp/googletest-gitclone.cmake:40 (message):
  Failed to checkout tag: 'master'
...
```

@Mizux could you take a look?

**Comment by toor1245** [comment]
https://github.com/google/cpu_features/pull/202

**Comment by Mizux** [comment]
Closed by #202 Thx again !!!
